backend/src/api/cycle.js

```diff
@@ -32,6 +36,37 @@ router.get(
   })
 )
 
+// get all cycles belonging to a workspace (either current, outdated, or both)
+// TODO: when authentication server is integrated, only show the cycles
+// to the workspace's owner. Otherwise, 404.
+router.get(
+  '/workspace/:workspaceName',
+  [directorsOnly],
+  errorWrap(async (req, res) => {
+    const workspaceName = req.params.workspaceName
+    const current = req.body.current
+
+    if (!workspaceName) {
+      res.json({
+        code: 400,
+        message: 'malformed request',
+        success: false
+      })
+    }
+
+    const cycles =
+      current !== null
+        ? await Cycle.find({ workspaceName, current })
+        : await Cycle.find({ workspaceName })
```

Comment:
super nitpicky but when reading this it ends up being a double negative for the `else` part